### DISCUSSION TITLE
Voting reminder

### DISCUSSION THREAD

**Discussion by Drakanoiderna** [Zrips/CMI#2239] [Enhancement]
**Description of issue or feature request:**

---Voting reminder have so eavry day a riminder to the players that havent voted to do so.

Thanks for a grate plugin

**Comment by mercurialmusic** [comment]
I wonder if you'd be able to use the `ExtraRewards:` section for this purpose. Just define a vote count of `'0'` and set it to run a command to remind them.

**Comment by Drakanoiderna** [comment]
Yes but wouldent that only be untill thy vote 1 time? I am looking for somthing that riminds then when thay havent vote on that day, , somthing that can run if the MaxVotesInADay is not true. Then the command runs 1 eavry login or xx time.
